src/getFolds.js

```diff
@@ -0,0 +1,37 @@
+// from CV.kfold ml-cross-validation
```

Comment:
can you remove it and use the ml-cross-validation package now?

Reply:
done